From b9e931eddc0fa6820300c4f4844eb5a0c2fdd045 Mon Sep 17 00:00:00 2001
From: Rob Clark <rclark@redhat.com>
Date: Wed, 17 Dec 2014 21:39:11 -0500
Subject: [redhat] Makefile: add RHEL_DRM_* version

Message-id: <1418852380-13061-61-git-send-email-rclark@redhat.com>
Patchwork-id: 102211
O-Subject: [RHEL7 drm 60/89] Makefile: add RHEL_DRM_* version
Bugzilla: 1173317
RH-Acked-by: Jerome Glisse <jglisse@redhat.com>
RH-Acked-by: Dave Airlie <airlied@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1173317
Upstream: n/a

It was a small request from virtualbox folks, for building out-of-tree
drm module.  While that isn't officially supported, this was a small
thing we could do to make their lives much easier.

Signed-off-by: Rob Clark <rclark@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Makefile b/Makefile
index 5db3915..cc6e775 100644
--- a/Makefile
+++ b/Makefile
@@ -6,6 +6,9 @@ NAME = Unicycling Gorilla
 RHEL_MAJOR = 7
 RHEL_MINOR = 1
 RHEL_RELEASE = 220
+RHEL_DRM_VERSION = 3
+RHEL_DRM_PATCHLEVEL = 16
+RHEL_DRM_SUBLEVEL = 7
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
-- 
1.7.1